app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ItemsController < ApplicationController
+    def show
+        items = Item.where(merchant_id: params[:id])
+        serialized_items = ItemSerializer.new(items)
+        render json: serialized_items
+    end 
+
```

Comment:
You should consider using a linter like Rubocop (https://github.com/rubocop-hq/rubocop). I am mentioning it here because of the extra whitespace, but it helps out a lot with figuring out how to better structure your code.

Steps to use Rubocop ->

`gem install rubocop`
`rubocop`

Rubocop Usage
Autocorrect many common things with `rubocop -a`

Reply:
Thanks Luke! I will add it in!